frontend/packages/dev-console/src/components/topology/shapes/WorkloadNode.tsx

```diff
@@ -1,10 +1,17 @@
 import * as React from 'react';
 import { PenIcon, ExternalLinkAltIcon } from '@patternfly/react-icons';
+import { Status } from '@console/shared';
+import { Link } from 'react-router-dom';
 import { NodeProps, WorkloadData } from '../topology-types';
 import Decorator from './Decorator';
 import BaseNode from './BaseNode';
 import PodStatus from './PodStatus';
 import KnativeIcon from './KnativeIcon';
+import BuildSuccessIcon from './BuildSuccessIcon';
+
+export type GreenOutlinedCheckCircleIconProps = {
```

Comment:
Left over code? Remove...

Reply:
removed in https://github.com/openshift/console/pull/2193/commits/778814f46cc371c46dfe4037cf52bb29b1a85082